python/cudf_polars/cudf_polars/dsl/utils/aggregations.py

```diff
@@ -24,6 +24,17 @@
 __all__ = ["apply_pre_evaluation", "decompose_aggs", "decompose_single_agg"]
 
 
+def _supports_sum(dtype: plc.DataType) -> bool:
```

Comment:
Maybe there's a libcudf helper for this?

Reply:
We do it's called https://docs.rapids.ai/api/cudf/stable/libcudf_docs/api_docs/aggregation_factories/#_CPPv420is_valid_aggregation9data_typeN11aggregation4KindE 😄